### PR TITLE
Merge duplicated implementations of Instant-to-String and String-to-Instant

### PR DESCRIPTION
**Pull request by dmikurube**
In `embulk-core` for some reasons, we have duplicated the conversion methods from `Instant` (`Timestamp`) to `String`, and `String` to `Instant` (`Timestamp`). But it's not good, of course. It's time to merge them into one implementation.

### CODE REVIEW

File: embulk-core/src/main/java/org/embulk/spi/time/TimestampSerDe.java [hunk 1]
```diff
@@ -9,66 +9,41 @@
...
     public static void configure(ObjectMapperModule mapper) {
         SimpleModule module = new SimpleModule();
-        module.addSerializer(Timestamp.class, new DateTimeZoneSerializer());
-        module.addDeserializer(Timestamp.class, new DateTimeZoneDeserializer());
+        module.addSerializer(Timestamp.class, new TimestampSerializer());
+        module.addDeserializer(Timestamp.class, new TimestampDeserializer());
         mapper.registerModule(module);
     }
 
-    public static class DateTimeZoneSerializer extends JsonSerializer<Timestamp> {
```
Comment (dmikurube) [inline embulk-core/src/main/java/org/embulk/spi/time/TimestampSerDe.java:26]: Historical mistake. `TimestampSerDe` has had `DateTimeZone(De)Serializer` in it...

Comment (shroman) [reply embulk-core/src/main/java/org/embulk/spi/time/TimestampSerDe.java:26]: I see...

File: embulk-core/src/test/java/org/embulk/spi/TestExecSession.java [hunk 2]
```diff
@@ -1,30 +1,5 @@
...
-        assertToStringFromString("2015-01-19 07:36:10.123456780 UTC", Instant.ofEpochSecond(1421652970, 123456780));
-        assertToStringFromString("2015-01-19 07:36:10.123456789 UTC", Instant.ofEpochSecond(1421652970, 123456789));
-    }
-
-    private void assertToStringFromString(final String expectedString, final Instant instant) {
-        final String actualString = ExecSession.toStringFromInstantForTesting(instant);
-        assertEquals(expectedString, actualString);
-        assertEquals(instant, ExecSession.toInstantFromStringForTesting(actualString).get());
-    }
+    // TODO: Implement tests to build ExecSession from ConfigSource.
```
Comment (shroman) [inline embulk-core/src/test/java/org/embulk/spi/TestExecSession.java:4]: maybe github issue too?

File: embulk-core/src/test/java/org/embulk/spi/time/TestTimestampSerDe.java [hunk 3]
```diff
@@ -1,25 +1,5 @@
...
-        checkToStringFromString(Timestamp.ofEpochSecond(1421652970, 123456 * 1000));
-        checkToStringFromString(Timestamp.ofEpochSecond(1421652970, 123456700));
-        checkToStringFromString(Timestamp.ofEpochSecond(1421652970, 123456780));
-        checkToStringFromString(Timestamp.ofEpochSecond(1421652970, 123456789));
-    }
-
-    private void checkToStringFromString(final Timestamp timestamp) {
-        Assert.assertEquals(timestamp, TimestampSerDe.createTimestampFromStringForTesting(timestamp.toString()));
-    }
+    // TODO: Implement tests of TimestampSerDe as Jackson serializer and deserializer.
```
Comment (shroman) [inline embulk-core/src/test/java/org/embulk/spi/time/TestTimestampSerDe.java:4]: adding github issue for this?
with `newbie` label for instance

**Comment by dmikurube** [comment]
Merging.

**Pull request merged**